packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageModule.kt

```diff
@@ -42,4 +56,28 @@ class ExpoImageModule(val context: ReactApplicationContext) : ReactContextBaseJa
       }
     }
   }
+
+  @ReactMethod
+  fun getSize(url: String, promise: Promise) {
+    try {
+      Glide
+          .with(context)
+          .`as`(ExpoImageSize::class.java)
+          .apply(sizeOptions)
+          .load(url)
+          .into(object : SimpleTarget<ExpoImageSize>() {
+            private val promiseRef = promise
```

Comment:
I'm not sure if you need this one? 🤔
Let's get rid of this if possible.

Reply:
I believe you refer to the unnecessary `promiseRef` property -- removed it.